README.md

```diff
@@ -1,3 +1,36 @@
-# Backtrace-Javascript
+<p align="center">
+    <a href="https://backtrace.io/" target="_blank">
```

Comment:
Backtrace logo color is weird. Are you sure this is a valid Backtrace logo?

Reply:
We cannot set a different image for light vs. dark mode. I constructed this one to handle both. Too weird? Not weird enough?

Reply:
Implemented a deprecated method to support light/dark images in GitHub. I expect it will continue to work fine.